packages/frontend/core/src/components/page-list/view/edit-collection/hooks.tsx

```diff
@@ -15,8 +16,11 @@ export const useSelectPage = ({
   const close = useCallback(() => {
     onChange(undefined);
   }, []);
-  return {
-    node: (
+
+  const { add } = useScopeRootComponents('select-page-modal');
```

Comment:
```suggestion
  const { mount } = useMount('select-page-modal');
```

Reply:
`useMount` seems a little vague, how about ﻿`useMountToNearestScope`?

Reply:
I think we'd better separate this from `FrameworkScope`. Sometimes other contexts also want to nest a mountpoint, so it's better not to include `scope` in the name.